services/system/HttpServer/src/HttpServer.cpp

```diff
@@ -93,12 +94,17 @@ namespace SystemService
       constexpr std::string_view allowedHeaders[] = {"Cache-Control",            //
                                                      "Content-Encoding",         //
                                                      "Content-Security-Policy",  //
-                                                     "ETag"};
+                                                     "ETag",                     //
+                                                     "Set-Cookie"};
```

Comment:
afaict you can eliminate all changes in this file except adding this Set-Cookie allowance